tools/serverpod_cli/lib/src/database/create_definition.dart

```diff
@@ -162,6 +162,6 @@ String? _getColumnDefault(
     case DefaultValueAllowedType.string:
       var defaultValue = column.defaultPersistValue;
       if (defaultValue == null) return null;
-      return defaultValue;
+      return '$defaultValue::text';
```

Comment:
Question: Is this what prevents this from being vulnerable to SQL injections?

Reply:
Fixed!

Reply:
I'm not sure this answers my question. How is it fixed? And is this what prevents SQL injections?